### DISCUSSION TITLE
Pipeline misconfiguration failures

### DISCUSSION THREAD

**Discussion by marcomattoni** [aquasecurity/trivy#8048] [Q&A]
### Question

Hello everyone! 

I am having issues with the CI/CD pipeline scanner in a GitHub repo. The following check IDs are coming up:

AVD-PIPELINE-0020, AVD-PIPELINE-0021, AVD-PIPELINE-0023, AVD-PIPELINE-0024, and AVD-PIPELINE-2025.

This appear in PR scans and are not exiting the workflow (the scan passes).

Any ideas of why or how can I fix it?

Thank you so much

### Target

Git Repository

### Scanner

Misconfiguration

### Output Format

None

### Mode

Client/Server

**Comment by simar7** [comment]
hi @marcomattoni are these custom checks? Trivy OSS does not have them.

**Reply by marcomattoni** [reply]
Hi @simar7! Thanks for the prompt response. 

Note: Newbie to Trivy and AquaSec here :) 

To answer your question: They are not. We are calling the --checks-bundle-repository=registry.aquasec.com/trivy-checks:1. 

I just read in another discussion that IaC misconfigurations are cached on the client side and we did ran to an initial issue with the policies when the PR took place. I am wondering if those results are cached and still coming up, even after we fixed it.

**Reply by simar7** [reply]
It'd be best for you to raise a service ticket with the Aqua commercial product in this case. The commercial offering has a dedicated support team that can help you.

**Reply by marcomattoni** [reply]
Thank you!
